openedx/core/djangoapps/credentials/utils.py

```diff
@@ -64,3 +64,34 @@ def get_user_program_credentials(user):
         programs_credentials_data = get_programs_for_credentials(user, programs_credentials)
 
     return programs_credentials_data
+
+
+def get_xseries_credentials(user):
```

Comment:
The utilities in this module are all very closely related, and I think they should be refactored/simplified. That doesn't need to happen now, but at the very least we need to stop tying the code so tightly to XSeries. XSeries is an edx.org product based on programs.

This function's job is to prepare program credential data for display. It should accept an optional category argument and use that to filter. Please also change the name of the function to remove explicit mention of XSeries.